.github/workflows/ci.yml

```diff
@@ -145,10 +145,14 @@ jobs:
           sudo time rm -rf /usr/local/lib/android
           sudo time rm -rf /usr/share/dotnet
           sudo time rm -rf /usr/share/swift
+          sudo time rm -rf /usr/local/share/powershell
```

Comment:
We could reuse [this](https://github.com/getsentry/sentry-unity/blob/e1c1e4476e83212f0daec856dd018ec4ca71d3f4/.github/workflows/ci.yml#L273-L281) action from the Unity SDK CI that helps us clear out the runner for more space.

Reply:
Somehow this action still misses some bits that we used to remove manually and the UE 5.4 container can't start due to a lack of disk space.